packages/dcos-net/extra/dcos-net-setup.py

```diff
@@ -30,38 +31,53 @@ def main():
         sys.stderr.buffer.write(result.stderr)
         if result.stderr.strip().endswith(b'File exists'):
             result.returncode = 0
-    elif sys.argv[1] == 'iptables':
+        sys.exit(result.returncode)
+
+    if sys.argv[1] == 'iptables':
         # check whether a rule matching the specification does exist
         argv = ['-C' if arg in ['-A', '-I'] else arg for arg in sys.argv[1:]]
         result = subprocess.run(argv)
         if result.returncode != 0:
             # if it doesn't exist append or insert that rules
             result = subprocess.run(sys.argv[1:])
-    elif sys.argv[1] == '--ipv6':
+        sys.exit(result.returncode)
```

Comment:
I would still inclined to keep the previous code.

Reply:
OK, let's keep the minimal change.